metatron-app/metatron-discovery
# account lock on repeated login failures

### Description

Account lock on repeated login failures

**Related Issue** : 



### How Has This Been Tested?

1. add password lock count config to application.yaml
   For testing, the count is 5.
```
polaris:
  user:
    password:
       lockCount: 5   # The Password must match within that number.
```
2. register a user.
3. approve user using admin.
4. login and logout
5. repeat 5 times in a row to try logging in with the wrong password.
6. Check the login fail message.
7. Reset the password using the 'Forgot password?' link on the login page.
8. Check if the password is unlocked using the temporary password.
9. Reset the password using member management page.
10. Check if the password is unlocked using the temporary password.


#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
